test/backend/openfisca/test.js

```diff
@@ -30,6 +39,38 @@ describe('openfisca generateTest', function() {
     });
 });
 
+function run_cmd(cmd, args, callback ) {
+    var spawn = require('child_process').spawn;
+    var child = spawn(cmd, args);
+    var respErr = "";
+    var respOut = "";
+
+    child.stdout.on('data', function (buffer) { respOut += buffer.toString(); });
+    child.stderr.on('data', function (buffer) { respErr += buffer.toString(); });
+    child.on('exit', function(code) { callback(code ? 'Exit code was ' + code : null, respOut, respErr); });
+    child.on('error', function(err) { callback(err); });
+}
+
+function runOpenFiscaTest(yaml, extension, done) {
```

Comment:
IMHO this should be in `backend/lib/openfisca/test.js`, and this shouldn't make any assertion. 
Assertions should be move inside the test (i.e, assert this command has returned zero exit code).

Reply:
Maybe we could return a Promise instead of using callback?

```
runOpenFiscaTest(yaml, extension)
  .then(stdout => console.log(stdout))
  .fail(e => console.log(e.code))
```